core/src/util/registry.rs

```diff
@@ -0,0 +1,114 @@
+#[cfg(target_os = "windows")]
```

Comment:
This cfg here is redundant with the one in mod.rs. I would say just leave the one in mod.rs (and presumably we'll have a module called ini for the other platforms).